lib/pages/share/share.dart

```diff
@@ -84,23 +84,52 @@ class ShareController extends State<Share>
       id: selectedRoomsNotifier.value.first,
       client: Matrix.of(context).client,
     );
+    final shareContentList = Matrix.of(context).shareContentList;
     final shareContent = Matrix.of(context).shareContent;
```

Comment:
remove `shareContent` if not needed, i see you remove it in `receive_sharing_intent_mixin.dart` already

Reply:
`shareContent` support to text, keep it. Only change for sharing files.